src/coalesce-vue-vuetify3/src/shared.scss

```diff
@@ -71,3 +71,10 @@
     max-height: calc(100% - 9px);
   }
 }
+
+.fixed-table-column-right {
+  position: sticky;
+  right: 0;
+  background: rgb(var(--v-theme-surface-light));
+  box-shadow: -2px 2px 4px 0px rgba(0, 0, 0, 0.4);
```

Comment:
Will need to add some JS to watch for scrolls so that this effect is only applied when the table has horizontal scrolling. It looks too weird on tables that don't have h scrolling.

![image](https://github.com/user-attachments/assets/aaa9f22c-15fc-446e-b852-479a7bf87b0e)

Reply:
Yes - I almost did that before making this PR... but decided to see if it'd be a worthwhile change first. So I'll go add that.